src/main/java/org/rundeck/plugins/S3LogFileStoragePlugin.java

```diff
@@ -348,6 +349,8 @@ protected boolean storePath(
         try {
             amazonS3.putObject(putObjectRequest);
             return true;
+        } catch (SdkClientException e){
+            throw new ExecutionFileStorageException(e.getMessage(), e);
```

Comment:
write on a log with level debug, showing the message like:
" _job could still be executing:_ " followed by the `e.getMessage()`